### ISSUE TITLE
responder.recv(&mut msg, 0).unwrap(); ERROR: expected struct `zmq::Message`, found enum `std::result::Result`

### ISSUE THREAD

**Issue by manhnt9** [erickt/rust-zmq#225]
I'm trying example [hello world server](https://github.com/erickt/rust-zmq/blob/master/examples/zguide/helloworld_server/main.rs) and get this compile error:
```
 --> src/main.rs:64:24
   |
64 |         responder.recv(&mut msg, 0).unwrap();
   |                        ^^^^^^^^ expected struct `zmq::Message`, found enum `std::result::Result`
   |
   = note: expected type `&mut zmq::Message`
              found type `&mut std::result::Result<zmq::Message, zmq::Error>`
```

**Comment by manhnt9** [comment]
change
```rust
let mut msg = zmq::Message::new();
```
to 
```rust
let mut msg = zmq::Message::new().unwrap();
```
fixes the error

**Comment by erickt** [comment]
Thanks! This has been fixed in the almost released 0.9.0.

**Comment by kp995** [comment]
Hi erickt, It seems like for the helloWorld_server and client files are not updated to what you said above. 

Its still 
`let mut msg = zmq::Message::new();`

https://github.com/erickt/rust-zmq/blob/master/examples/zguide/helloworld_server/main.rs#L18

Can you please fix this. Thanks!
